plugins/browser-plugin-error-tracking/src/index.ts

```diff
@@ -74,7 +75,7 @@ export function trackError(
   trackers: Array<string> = Object.keys(_trackers)
 ) {
   const { message, filename, lineno, colno, error, context, timestamp } = event,
-    stack = error && error.stack ? error.stack : null;
+    stack = error && truncateStackTrace(error.stack);
```

Comment:
Can you please add a unit test to make sure we don't regress on the behaviour in the future?